x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/FeatureNotEnabledException.java

```diff
@@ -29,6 +29,7 @@ public enum Feature {
         }
     }
 
+    @SuppressWarnings("this-escape")
```

Comment:
Not sure why this warning got reported now. Maybe because I've added new classes to the same package? 🤔